.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -192,6 +192,24 @@ steps:
 
         rm -f new_test_plan_id.txt
 
+        # all feature branches branched out from master can use py3only image
+        parent_commit=$(git rev-parse HEAD^1)
+        master_commit=$(git rev-parse master)
+        branch_from_master=0
+        if [ "$parent_commit" == "$master_commit" ]
+        then
+          branch_from_master=1
+        fi
+
+        if [ "${{ parameters.MGMT_BRANCH }}" == "master" ] || [ $branch_from_master -eq 1 ]
+        then
+          PTF_IMAGETAG="--ptf_imagetag=py3only"
+        else
+          PTF_IMAGETAG="--ptf_imagetag=internal"
+        fi
+        common_extra_params="${{ parameters.COMMON_EXTRA_PARAMS }} $PTF_IMAGETAG"
+        echo "##vso[task.setvariable variable=COMMON_EXTRA_PARAMS;]$common_extra_params"
```

Comment:
This is to create pipeline variable "COMMON_EXTRA_PARAMS". Then subsequent pipeline tasks can access to this pipeline variable.

Since the common extra params is only used this task, it is unnecessary to create such pipeline variable. Just use bash variable should be enough. Then, in line 223, refer to bash variable instead of pipeline parameter.

Reply:
Thank you. I have modified that bit.